docs/site/index/survey-tool/managing-users.md

```diff
@@ -14,68 +14,83 @@ Open the survey tool, and click on the gear icon in the top left, then pick **Li
 ## To add a user
 
 Click on \[Add User] at the top:
-![image](../../images/index/managingUsers1.png)
 
-➡️
+![image](../../images/index/managingUsers1.png)
 
 ![image](../../images/index/managingUsers2.png)
 
-Set the name, email, and locales responsible. To see the locale codes, go to http://unicode.org/cldr/apps/survey?p_codes=t. Just use the language codes in on the right side\-\- the country parts are ignored.
-- *Note that you must use the right code in the case of synonyms: use he* ***not*** *iw, use pt* ***not*** *pt\_BR...*
+Set the name, email, and the locale(s) they are responsible for.
 
 ![image](../../images/index/managingUsers3.jpg)
 
 Set the Userlevel based on your assessment of the person's skill. Normally this is "Vetter". You can use "Street" for volunteers whom you don't know. People don't have to come from your organization as long as you have confidence in them.
+
 - See the [CLDR Process](/index/process) page for a description of the user levels and their privileges.
 
-*Known bug: if you try to create a user when the user already has an account, you get an unknown error message. Often this is because the user is locked. Click on "Show locked users", find the person, and change the status (see below).*
+*Known bug: if you try to create a user and that email address already has an account, you get an unknown error message. Often this is because the user is locked. Click on "Show locked users", find the person, and change the status (see below).*
+
+## To forward old votes to a new user
+When replacing a previous user with a new one, you will likely want that new user to inherit the votes from their predecessor. Otherwise the new user would have to submit all votes from scratch manually, which can be several thousands.
+
+1. Once the new user has been created, find them in the list of users and click on the link "Copy votes to this vetter"
+2. Choose the user's predecessor (i.e. the user you want to copy votes *from*).
+3. Choose what locale you want to copy votes *from*. (In case the old user has been active in multiple locales.)
+4. Choose what locale you want to copy the old user's votes *to*. (In almost all cases, that will be the same locale, which is the default choice.)
+5. Vote copying will take place.
+6. Tell the new user to exit and re-enter the survey tool.
+
+At this time, the survey tool will consider the old user's voting history the same as the new user's. When the new user logs in, survey tool will automatically apply all the old user's votes **when they match the currently shipping value**. In any case where the old vote does not match the currently shipping value OR where the value is new, the vote will be abstained and the new user will have to submit a new vote.
+
+Remind the new user that **they are still responsible for inherited votes**, and should review them carefully to ensure they agree with them. The vote inheritance is provided for efficiency and convenience, but does not shift responsibility.
```

Comment:
Should we also mention that votes can be imported from another organization?

Reply:
Can they? I don't know that they can from within the confines of the survey tool though? Either way, it seems like an unusual case.

Reply:
@srl295 In discussion, I thought you said that a Manager of organization X could import votes to user A from a user B that is in organization Y, in the Survey tool. Is that true?

Reply:
that sounds more like _moving_ user from organization Y to organization X.  Probably an admin can copy votes between organization.  But I think you have to have permissions to manage both A and B to do the copy.